wiki/Contests/Official_support/en.md

```diff
@@ -0,0 +1,66 @@
+---
+tags:
+  - badge
+  - badges
+  - badged
+---
+
+# Official contest support
+
+[Beatmapping contests](/wiki/Contests) are eligible to recieve support from the [osu! team](/wiki/People/The_Team) provided they meet certain criteria.
```

Comment:
The title of this article is for contests in general, so limiting the article to just beatmap contests feels weird. I know that other types of contests are very rare, however a note about these should be left here. As someone who hosts other kinds of contests myself I know how terrible the wiki already is at guiding people like me, so this would be a good opportunity to address that.